code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm

```diff
@@ -176,6 +176,8 @@
 	xeno_cooldown = 13 SECONDS
 	plasma_cost = 80
 
+	var/click_miss_cooldown = 10
```

Comment:
defining these without using them won't do anything - the warrior implementation contains a check in /lunge/use_ability() that actually applies this cooldown

![image](https://github.com/user-attachments/assets/01b8870c-2a56-4986-b021-1fd6cec11418)